### PR TITLE
Animation

### PR DESCRIPTION
**Pull request by ptpham**

### CODE REVIEW

File: src/paper.js [hunk 1]
```diff
@@ -348,10 +348,20 @@ Paper.prototype.equivalentTo = function(other) {
   var first = this.toJSON(), second = other.toJSON ? other.toJSON() : other;
   return first.size == second.size
     && geometry.equivalent(first.faces, second.faces)
+    && geometry.equivalent(first.vertices, second.vertices)
     && geometry.equivalent(first.points, second.points)
     && geometry.equivalent(first.angles, second.angles)
     && geometry.equivalent(first.naturals, second.naturals);
 };
 
+Paper.vertexify = function(faces, vertices) {
```
Comment (cixzhang) [inline src/paper.js]: Teest? :D

File: test/triangulate.js [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+  it('fans faces with more than 3 values', function () {
     var face = [1, 2, 3, 4, 5];
-    expect(triangulate(face)).to.deep.equal([1, 2, 3, 1, 3, 4, 1, 4, 5]);
+    expect(triangulate.fan(face)).to.deep.equal([1, 2, 3, 1, 3, 4, 1, 4, 5]);
+  });
+  
+  it('can generate mirrors for triangles', function() {
+    var triangles = [[0,1,2],[2,1,3]];
+    var result = triangulate.generateMirrors(triangles);
+    var expected = [[[3,1],null,null],[null,null,[0,0]]];
```
Comment (cixzhang) [inline test/triangulate.js]: What are these mirrors?

File: src/paper.js (hunk 1 above)
Comment (ptpham) [inline src/paper.js]: I ... I have a test in a later branch? XD T___T

File: test/triangulate.js (hunk 2 above)
Comment (ptpham) [inline test/triangulate.js]: I'll add a comment in a later revision. XD

**Pull request merged**